config_docker.yml

```diff
@@ -9,19 +9,12 @@ sql:
       database: 'mimic'
       user: 'mimicuser'
       password: 'mimicuser'
-  fhirbase:
-    lib: 'psycopg2'
-    args: []
-    kwargs:
-      host: 'fhir-pipe-fhirbase'
-      port: 5432
-      database: 'fhirbase'
-      user: 'postgres'
-      password: 'postgres'
-  oracle:
```

Comment:
Same here I think we want to keep oracle

Reply:
Done.